tests/test_normalize.py

```diff
@@ -84,7 +84,7 @@ def test_normalize_standardize_allsamples():
     Testing normalize pycytominer function
     method = "standardize"
     meta_features = None
-    samples="all"
+    samples="all".
```

Comment:
These changes which add the period at the end of method specification felt a little off, and made me wonder about the value-add of these changes. I'd suggest rolling back these changes unless there's a compelling reason to keep them. I also wasn't sure more generally: was the ruff configuration supposed to ignore testing docstrings?